src/lib.rs

```diff
@@ -109,6 +109,13 @@ impl<W: HasRawWindowHandle> GraphicsContext<W> {
     }
 }
 
+impl<W: HasRawWindowHandle> AsRef<W> for GraphicsContext<W> {
+    /// Equivalent to [`self.window()`](Self::window()).
+    fn as_ref(&self) -> &W {
```

Comment:
Is there any particular reason that this isn't `#[inline]`?

Reply:
No; done.

This may be redundant since the function is generic and generic functions are, in the current Rust compiler, always inlinable, but it makes some sense to _explicitly_ follow the general recommended practice of making trivial accessor functions inlinable.